src/v/kafka/server/usage_aggregator.cc

```diff
@@ -109,16 +109,34 @@ usage& usage::operator+=(const usage& other) {
     return *this;
 }
 
-template<typename time_point>
-auto epoch_time_secs(time_point now) {
+/// The type that will be providing timestamps in form of seconds since
+/// epoch. ss::lowres_clock has no concept of epoch time, therefore it isn't
```

Comment:
even the lowres clock has an epoch value, it's just not defined as the unix epoch, and is probably more like time since boot.